### PR TITLE
Change javascript action to vue action

### PR DESCRIPTION
**Pull request by moiikana**
does this look too hacky? is the data function right at this point?

### CODE REVIEW

File: skins/wmde19/src/components/layout/Header.vue [hunk 1]
```diff
@@ -1,54 +1,48 @@
...
-			</div>
+    <div class="navbar is-fixed-top has-background-bright has-shadow">
+        <div class="container">
+            <div class="navbar-brand">
+                <a class="navbar-item" href="/">
+                    <img :src="imagePath + '/images/logo-horizontal-wikimedia.svg'" alt="Wikimedia Deutschland"
+                         width="144" height="29">
+                </a>
+                <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false"
+                   v-bind:class="{ 'is-active': showNavbarBurger }" v-on:click="showNavbarBurger = !showNavbarBurger">
```
Comment (tzhelyazkova) [inline skins/wmde19/src/components/layout/Header.vue]: you can list all classes together in the vue directive like
`v-bind:class="[{ 'is-active': showNavbarBurger }, 'navbar-burger']"`
Also `v-on:click` can be `@click`

File: skins/wmde19/src/components/layout/Header.vue [hunk 2]
```diff
@@ -1,54 +1,48 @@
...
-					</a>
-					<a class="navbar-item">
-						Spenden
-					</a>
-				</div>
-			</div>
-		</div>
-	</div>
+            <div class="navbar-menu" id="navMenu"
+                 v-bind:class="{ 'is-active': showNavbarBurger }" v-on:click="showNavbarBurger = !showNavbarBurger">
```
Comment (tzhelyazkova) [inline skins/wmde19/src/components/layout/Header.vue]: Same for this `div`

File: skins/wmde19/src/components/layout/Header.vue [hunk 3]
```diff
@@ -1,54 +1,48 @@
...
-		$navbarBurgers.forEach( el => {
-			el.addEventListener( 'click', () => {
-				el.classList.toggle( 'is-active' );
-				document.querySelector('.navbar-menu').classList.toggle('is-active');
-			} );
-		} );
-	}
-} );
-
+	export default {
```
Comment (gbirke) [inline skins/wmde19/src/components/layout/Header.vue]: If you're feeling adventurous, have a peek at other components and do a `<script type="ts">` and see what happens and how you need to modify this component to become a TypeScript component.

**Comment by tzhelyazkova** [comment]
> Please restore the package.json and packa-lock.json files, they should not shop up in this commit. There are many ways to do this, my approach would be
> 
> ```
> git checkout HEAD^ skins/10h16/package-lock.json
> git checkout HEAD^ skins/10h16/package.json
> # more checkouts for the other files here ...
> git add skins/*/package*.json
> git commit --amend
> ```
> 

It's kinda my fault. I gave @moiikana wrong advice. 
The idea was to revert the changes in the `package(-lock)?.json` files as they are not relevant for this change. You still might wanna do that with `git reset HEAD .` and then adding only `Header.vue` and amending the commit.
Git is not easy 😞

**Comment by gbirke** [comment]
Merging because the failing CI is unrelated to this. Fixed in #1469

**Pull request merged**